docs/release-notes.rst

```diff
@@ -3,6 +3,31 @@ Release notes
 
 .. towncrier release notes start
 
+3.0.5 (2025-03-07)
+------------------
+
+Features
+~~~~~~~~
+
+- Adds functions for concurrently creating multiple arrays and groups. (:issue:`2665`)
+
+
+Bugfixes
+~~~~~~~~
+
+- Fixed a bug where ``ArrayV2Metadata`` could save ``filters`` as an empty array. (:issue:`2847`)
+- Fixed a bug where ``StorePath`` creation would not apply standard path normalization to the ``path`` parameter,
+  which led to the creation of arrays and groups with invalid keys. (:issue:`2850`)
+- Fix a bug when setting values of a smaller last chunk. (:issue:`2851`)
```

Comment:
```suggestion
- Fix a bug when setting values of a smaller last chunk. (:issue:`2851`)
```

this should've been in the previous release I guess

Reply:
now included in the previously missing 3.0.4 section